rpmlint/checks/FileDigestCheck.py

```diff
@@ -102,20 +103,24 @@ def _calculate_errors_for_digest_group(self, pkg, digest_group, secured_paths):
         # report errors for secured files not covered by the digest group
         for filename in secured_paths - covered_files:
             group = self._get_digest_configuration_group(pkg.files[filename])
-            errors.append((f'{group}-file-digest-unauthorized', filename))
+            errors.append((f'{group}-file-digest-unauthorized', filename, None))
 
         # report errors for missing files mentioned in the digest group
         for filename in covered_files - pkg_files:
             group = self._get_digest_configuration_group(pkg.files[filename])
-            errors.append((f'{group}-file-digest-unauthorized', filename))
+            errors.append((f'{group}-file-digest-unauthorized', filename, None))
 
         # report errors for invalid digests
         for digest in digest_group:
             filename = digest['path']
             if filename in pkg_files:
                 group = self._get_digest_configuration_group(pkg.files[filename])
-                if not self._is_valid_digest(pkg.files[filename], digest, pkg):
-                    errors.append((f'{group}-file-digest-mismatch', filename))
+                valid_digest, file_digest = self._is_valid_digest(pkg.files[filename], digest, pkg)
+                if not valid_digest:
+                    error_detail = None
+                    if file_digest:
+                        error_detail = f'expected:{digest["hash"]}, has:{file_digest}'
```

Comment:
Could you please also add the digest algorithm used to the output? So it is clear to everybody which kind of digest this is.